src/test/java/htsjdk/samtools/util/IntervalTreeTest.java

```diff
@@ -57,17 +60,100 @@ private int countElements(final Iterator<IntervalTree.Node<String>> it) {
         return ret;
     }
 
-    @Test
-    public void testMatches()
-    {
-        final IntervalTree<String> intervalTree = new IntervalTree<String>();
+    private final IntervalTree<String> intervalTree = new IntervalTree<String>();
+
+    @BeforeMethod
+    public void init(){ //due to the destructive nature of removeMany test...
```

Comment:
I'm assuming that this is due to concernes regarding multithreading. I'll add `@Test(singleThreaded=true) ` to the class